packages/lake/src/assets/fonts/RobotoMono.css

```diff
@@ -0,0 +1,7 @@
+@font-face {
+  font-family: "Roboto Mono";
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
```

Comment:
As the font is mandatory, it's probably better to remove `font-display: swap`

Reply:
I set block because I prefer to use explicit value
https://github.com/swan-io/lake/pull/93/commits/3f9d52f8bf4067206c118d2392db7d9aff9b61bb